caravel/templates/index.html

```diff
@@ -1,28 +1,43 @@
 {% extends "layout.html" %}
 {% block title %}Listings{% endblock %}
 {% block content %}
-  <div class="row">
-    {% for listing in listings %}
-    <div class="listing col-xs-12 col-md-3">
-      <h3 class="title">
-        <a href="/{{ listing.key }}?q={{ request.args['q']}}">
-          {{ listing.title }}
-        </a>
-      </h3>
-      <p class="price">{{ '${:,.2f}'.format(listing.price / 100) }}</p>
+  {% set i = 0 %}
+  {% for listing in listings %}
+    {% set j = i + 1 %}
+    {%  if i%4 == 0 %}
+      <div class="row row-eq-height">
+    {% endif %}
+    <div class="col-xs-12 col-md-3" style="margin-left: 10px; margin-right: 10px; margin-bottom:15px; padding:0; background-color: #ffffff;
```

Comment:
Fixing constant height windows. But requires using the row div for every 4 cols. 

Planed to add or remove those tags based on responsible JS. Hence the default.js file.